### ISSUE TITLE
Provider Settings EnergyZero

### ISSUE THREAD

**Issue by c0mplex1** [TheFes/cheapest-energy-hours#115]
I use these parameters to retrieve the electricity rates from EnergyZero:
`attr_all = "Prices", time_key = "readingDate"`

**Comment by TheFes** [comment]
Which integration do you use for this?

**Comment by c0mplex1** [comment]
I created my own integration and you helped me to simplify the code
https://community.home-assistant.io/t/cheapest-energy-hours-jinja-macro-for-dynamic-energy-prices/569370/108

I think you can also use the HA integration from @klaasnicolaas (Klaas Schoute), but I'm not for sure.
https://www.home-assistant.io/integrations/energyzero/

**Comment by klaasnicolaas** [comment]
Yes, you can retrieve the hourly data from EnergyZero via a service with a template trigger (described in the docs).

**Comment by TheFes** [comment]
Ah, you're using a rest sensor. You could also use the service call which was added.

**Comment by TheFes** [comment]
I've added how to create a sensor, and use it (using the service call) to the [docs](<https://github.com/TheFes/cheapest-energy-hours/blob/main/documentation/1-source_sensor.md#data-provider-settings>)
